### DISCUSSION TITLE
Installing many languages at once command, like npm

### DISCUSSION THREAD

**Discussion by Yohnny777** [nvim-treesitter/nvim-treesitter#4094] [Q&A] [answered]
Is there a good way to do this? Let's say I have 10 languages to install, do I have to `:TSInstall typescript` ... or is there a more handy way to do this?

Ideally I was thinking of `:TSInstall html css javascript typescript ...` command

**Comment by clason** [comment, marked as answer]
Did you not try that?

**Reply by Yohnny777** [reply]
I did and something went wrong then, I must've assumed this syntax is not supported. It indeed works now, thanks.
